connector/src/main/java/org/geysermc/connector/utils/MathUtils.java

```diff
@@ -60,6 +60,26 @@ public static double constrain(double num, double min, double max) {
         return num;
     }
 
+    /**
+     * If number is greater than the max, set it to max, and if number is lower than low, set it to low.
+     * @param num number to calculate
+     * @param min the lowest value the number can be
+     * @param max the greatest value the number can be
+     * @return - min if num is lower than min <br>
+     * - max if num is greater than max <br>
+     * - num otherwise
+     */
+    public static int constrain(int num, int min, int max) {
+        if (num > max) {
+            num = max;
+        }
+        if (num < min) {
```

Comment:
```suggestion

        if (num < min) {
```

Reply:
This also affects the constrain method for doubles in the same file. I'm fine with applying it here, but it should then be applied to the other function too while we're at it.